File: evidence/generated-systems/golang/aerospace-on-golang/aerospace-on-golang/internal/model/ProductionCertificate.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// ProductionCertificate Declaration
//==============================================================
type ProductionCertificate struct {
    gorm.Model
     CertificateNumber                                    string
    Authority                                    string
    ManufacturerId         *uint
    Manufacturer           *AerospaceManufacturer `gorm:"foreignKey:ManufacturerId"`

// parent associations as their child

}

File: evidence/generated-systems/django/aerospace-on-django/aerospaceOnDjango/models/ProductionCertificate.py
from django.db import models

#======================================================================
# 
# Encapsulates data for model ProductionCertificate
#
# @author Harbormaster Dev Team
#
#======================================================================

#======================================================================
# Class ProductionCertificate Declaration
#======================================================================
class ProductionCertificate (models.Model):

#======================================================================
# attribute declarations
#======================================================================
	certificateNumber = models.CharField(max_length=200, null=True)
	authority = models.CharField(max_length=200, null=True)
	manufacturer = models.ForeignKey('AerospaceManufacturer', on_delete=models.CASCADE, null=True, blank=True, related_name='+')

#======================================================================
# function declarations
#======================================================================
	def toString(self):
		str = ""
		str = str + self.certificateNumber
		str = str + self.authority
		return str;
    
	def __str__(self):
		return self.toString();

	def identity(self):
		return "ProductionCertificate";
    
	def objectType(self):
		return "ProductionCertificate";
